backend/api/health.py

```
@@ -141,51 +221,38 @@ async def envelope_replay(event_id: str):
-        avatars.append({
-            "id": agent_id,
-            "name": name,
-            "role": role,
-            "position": WORLD_POSITIONS[index],
-            "model": model,
-            "state": state,
-            "activity_summary": activity_summary,
-            "last_event_at": last_event_at,
-        })
-
+def lounge_state(db: Session = Depends(get_db)):
```

Ruff (0.16.2)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)